### PR TITLE
Db contrib/waltz 3758 provide export of servers dbs eol ubr v

### PR DESCRIPTION
**Pull request by xLcc**
#3758

### CODE REVIEW

File: waltz-ng/client/technology/components/group-technology-summary.html [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <div>
+    <div class="pull-right">
+        <waltz-data-extract-link name="Export"
+                                 styling="link"
+                                 extract="technology-server/{{$ctrl.parentEntityRef.id}}"
```
Comment (davidwatkins73) [inline waltz-ng/client/technology/components/group-technology-summary.html]:
```suggestion
                                 extract="technology-server/{{$ctrl.parentEntityRef.kind}}/{{$ctrl.parentEntityRef.id}}"
```

File: waltz-web/src/main/java/com/khartec/waltz/web/endpoints/extracts/TechnologyEOLDatabaseExtractor.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    private static final Logger LOG = LoggerFactory.getLogger(TechnologyEOLDatabaseExtractor.class);
+
+    public TechnologyEOLDatabaseExtractor(DSLContext dsl) {
+        super(dsl);
+    }
+
+
+    @Override
+    public void register() {
+        String path = mkPath("data-extract", "technology-database", ":id");
```
Comment (davidwatkins73) [inline waltz-web/src/main/java/com/khartec/waltz/web/endpoints/extracts/TechnologyEOLDatabaseExtractor.java]:
```suggestion
        String path = mkPath("data-extract", "technology-database", ":kind", ":id");
```

**Comment by rovats** [comment]
> Needs to be aware of other contexts, not just org-unit.

Probably need to use app-id selector factory?

**Comment by davidwatkins73** [comment]
y, re-working it with Ashley.  Will be using `appIdSelectorFactory` and also the dump straight to csv option in jOOQ (`.fetch().formatCSV()`)

**Pull request merged**